frontend/packages/dev-console/src/components/edit-application/edit-application-utils.ts

```diff
@@ -239,6 +239,7 @@ export const getUserLabels = (resource: K8sResourceKind) => {
     'app.kubernetes.io/part-of',
     'app.openshift.io/runtime-version',
     'app.openshift.io/runtime-namespace',
+    'app.openshift.io/runtime-icon',
```

Comment:
Please revert this change to use `runtime` label. runtime label itself points the imagestream icon.
```suggestion
    'app.openshift.io/runtime',
```

Reply:
Updated.